lib/src/chunksize.dart

```diff
@@ -0,0 +1,16 @@
+part of tiled;
+
+class Chunksize {
```

Comment:
can we rename everything to be camelCase or TitleCase? like ChunkSize, EditorSetting, offsetX, etc. I don't think this affects parsing and it's easier to read.

Reply:
Done